src/libs/actions/Report.js

```diff
@@ -1083,6 +1066,39 @@ function readNewestAction(reportID) {
         });
 }
 
+/**
+ * Gets the actions from the oldest unread action.
+ *
+ * @param {Number} reportID
+ * @param {Number} offset
+ */
+function readOldestAction(reportID, offset = 0) {
+    const hasFinishedLoadingData = {
+        onyxMethod: 'merge',
+        key: `${ONYXKEYS.IS_LOADING_REPORT_ACTIONS}${reportID}`,
```

Comment:
Looks kind of right, but there's a more correct way to do this. 

If we want to start tracking the loading status for each report in this way we would want to create a "collection" key. Explained in the features of Onyx [here](https://github.com/expensify/react-native-onyx#features), but let me know if you have any questions about it.

Reply:
So I would need to create a key in collections, and use that one instead? is that correct?